datafusion-cli/Dockerfile

```diff
@@ -21,8 +21,12 @@ COPY ./datafusion /usr/src/datafusion
 
 COPY ./datafusion-cli /usr/src/datafusion-cli
 
+COPY ./ballista /usr/src/ballista
+
 WORKDIR /usr/src/datafusion-cli
 
+RUN rustup component add rustfmt
```

Comment:
what's the reason for adding rustfmt here?

Reply:
oh, yeah, otherwise, it tells me 

```
#12 71.22    Compiling futures v0.3.16
#12 71.36    Compiling tonic-build v0.5.2
#12 71.71    Compiling bstr v0.2.16
#12 71.74    Compiling lz4 v1.23.2
#12 72.51    Compiling csv v1.1.6
#12 72.54    Compiling arrow-flight v5.2.0
#12 72.83    Compiling ballista-core v0.6.0 (/usr/src/ballista/rust/core)
#12 74.06    Compiling prettytable-rs v0.8.0
#12 74.58    Compiling arrow v5.2.0
#12 75.26 error: failed to run custom build command for `ballista-core v0.6.0 (/usr/src/ballista/rust/core)`
#12 75.26 
#12 75.26 Caused by:
#12 75.26   process didn't exit successfully: `/usr/src/datafusion-cli/target/release/build/ballista-core-702be6a42aa8bd82/build-script-build` (exit status: 1)
#12 75.26   --- stdout
#12 75.26   cargo:rerun-if-env-changed=FORCE_REBUILD
#12 75.26   cargo:rerun-if-changed=proto/ballista.proto
#12 75.26 
#12 75.26   --- stderr
#12 75.26   error: 'rustfmt' is not installed for the toolchain '1.54.0-x86_64-unknown-linux-gnu'
#12 75.26   To install, run `rustup component add rustfmt`
#12 75.26 warning: build failed, waiting for other jobs to finish...
#12 130.4 error: build failed

```

Reply:
Ha, good to know, I didn't know tonic_build depends on rustfmt by default for code gen formatting :)